packages/core/src/index.ts

```diff
@@ -210,6 +214,37 @@ export const create = (config: Config): VoiceCompass => {
 
   resetCallTimeout();
 
+  interface EscalationButtonProps {
+    wrap: HTMLElement | string,
+    text: string,
+  }
+
+  const appendEscalationButton = ({ wrap, text }: EscalationButtonProps) => {
+    if (!text)
+      throw new Error("Text isn't specified");
+
+    if (!wrap)
+      throw new Error("Wrapper element isn't specified or wasn't found");
```

Comment:
Runtime errors will crash the user's browser, just `console.error` and return from the function. Users will misuse it and complain that our scripts are crashing the rest of their page.